recipe/meta.yaml

```diff
@@ -1,11 +1,13 @@
+{% set version="1.0.2" %}
+
 package:
   name: gitpython
-  version: "1.0.1"
+  version: {{ version }}
 
 source:
-  fn: GitPython-1.0.1.tar.gz 
-  url: https://pypi.python.org/packages/source/G/GitPython/GitPython-1.0.1.tar.gz
-  md5: 4659644b42c41e5e5170c0cd83ed6422
+  fn: GitPython-{{ version }}.tar.gz
+  url: https://github.com/gitpython-developers/GitPython/archive/{{ version }}.tar.gz
+  sha256: 85de72556781480a38897a77de5b458ae3838b0fd589593679a1b5f34d181d84
```

Comment:
Borrowed the cleanup changes from `master`. So PR ( https://github.com/conda-forge/gitpython-feedstock/pull/6 ) in other words.